docs/root/intro/version_history.rst

```diff
@@ -3,6 +3,7 @@ Version history
 
 1.13.0 (pending)
 ================
+* http: added the ability to sanitize headers nominated by the Connection header. This new behavior is guarded by envoy.reloadable_features.connection_header_sanitization which defaults to true.
```

Comment:
mind moving this down?  we try for alphabetic order here

Reply:
Will do.

Reply:
Regarding the X-Forwarded for headers.  I added a comment below the existing one.  I will replace the lines on L445-446 with the updated comment.

See you at EnvoyCon! :)